common/SC_Win32Utils.h

```diff
@@ -64,6 +64,10 @@ int win32_pipe(int handles[2]);
 int win32_piperead(int s, char* buf, int len);
 int win32_pipewrite(int s, char* buf, int len);
 
+// missing Windows implementations in nova_tt
+bool win32_thread_set_affinity(int i);
+int win32_name_thread(const char* name);
```

Comment:
Yes, I just did this because it's the easiest and fastest solution. Of course, I can also make a PR to nova-tt. If Tim accepts it, it would be trivial to update the code.

> In general, I wouldn't mind eventually moving nova-tt functionality to the main codebase

I think it makes sense to keep it as an external library. We could use our own patched version, though, similar to portaudio. But that's a different discussion :-)

Reply:
If we intend to keep using using nova-tt, I think it would be good to make a PR there and then make the changes here (either in or after this PR, depending on timing).

Reply:
i'm happy to merge changes to nova-tt

Reply:
@timblechmann cool! will do a PR in a few days.